### PR TITLE
one typo; one misleading comment fixed

### PR DESCRIPTION
**Pull request by lure23**
## Thank you for your contribution!

We appreciate the time and effort you've put into this pull request.
To help us review it efficiently, please ensure you've gone through the following checklist:

### Submission Checklist 📝

- [x] I have updated existing examples or added new ones (if applicable).
- [~] I have used `cargo xtask fmt-packages` command to ensure that all changed code is formatted correctly.
- [-] My changes were added to the [`CHANGELOG.md`](https://github.com/esp-rs/esp-hal/blob/main/esp-hal/CHANGELOG.md) in the **_proper_** section.
- [-] I have added necessary changes to user code to the [Migration Guide](https://github.com/esp-rs/esp-hal/blob/main/esp-hal/MIGRATING-0.21.md).
- [x] My changes are in accordance to the [esp-rs API guidelines](https://github.com/esp-rs/esp-hal/blob/main/documentation/API-GUIDELINES.md)

*They are just 2 comment changes*

#### Extra:

- [x] I have read the [CONTRIBUTING.md guide](https://github.com/esp-rs/esp-hal/blob/main/documentation/CONTRIBUTING.md) and followed its instructions.

#### Description

Two unrelated lines, where I've noticed a typo, or an inconsistency. I've noticed good progress in simplifying the pins usage, and decided now is a nice time to let you know about these. 

#### Testing

>Describe how you tested your changes.

No testing, since only comments involved.

### CODE REVIEW

File: esp-hal/src/timer/systimer.rs [hunk 1]
```diff
@@ -49,7 +49,7 @@
 //!
 //! let mut systimer = SystemTimer::new(peripherals.SYSTIMER);
 //!
-//! // Get the current timestamp, in microseconds:
+//! // Get the current timestamp, in ticks of 16MHz (1/16 us):
```
Comment (bugadani) [inline esp-hal/src/timer/systimer.rs:52]: I don't think we want to meantion the actual value, as I don't think it's correct for the S2 or for different chrystals. We should be more clear that the actual tick rate can be accessed via `SystemTimer::ticks_per_second()`.

Comment (lure23) [reply esp-hal/src/timer/systimer.rs:52]: The point I have with this, is that the value is not "microseconds". Can change the text to be vaguer.

I noticed this when trying different approaches for `defmt` timing output. Ended up using `esp_hal::time::now` (which I think uses this one, underneath, but has wider MCU compatibility).

Comment (bugadani) [reply esp-hal/src/timer/systimer.rs:52]: I didn't say you are mistaken to change this :) Just that the result isn't perfect either.

> I noticed this when trying different approaches for defmt timing output.

Have you considered enabling [embassy-time/defmt-timestamp-uptime-s](https://docs.rs/crate/embassy-time/latest/features)? Or am I misunderstanding what you're trying to do?

Comment (lure23) [reply esp-hal/src/timer/systimer.rs:52]: Had a fresh look at the text in its context. How about:

```
Get the current tick count:
```

I think using `timestamp` makes one think in terms of `us`. There's a mention to `SystemTimer::ticks_per_second` just a few lines below, so I think this.. reduction in text would work sufficiently great.

>Have you considered enabling [embassy-time/defmt-timestamp-uptime-s](https://docs.rs/crate/embassy-time/latest/features)?

I haven't. I am on my way to Embassy, but not there yet. Will check.

Comment (bugadani) [reply esp-hal/src/timer/systimer.rs:52]: > Get the current tick count:

That sounds good to me

Comment (lure23) [reply esp-hal/src/timer/systimer.rs]: Changed; let me know if I am expected to do something more on this.

**Review by bugadani — APPROVED** [review]
Thank you

**Review by SergioGasquez — APPROVED** [review]
LGTM! Thanks

**Pull request merged**